Enterprise-CMCS/managed-care-review
# Move proto fields to contract and rate tables

## Summary

- This work moves all `Proto` form data fields into the `Postgres` tables.
- In preparation for the work in [MR-3301](https://qmacbis.atlassian.net/browse/MR-3301), I added a new field, `latestStateRateCertNumber` in the `State` table. This field function is to keep track of the rate certification number, exactly like how `latestStateSubmissionNumber` keeps track of submission numbers.

#### Related issues

#### Screenshots

#### Test cases covered



## QA guidance

- Please look at the tables and ensure the relationships make sense.
- Much of the code in the new `Postgres` functions are placeholders so that tests still run with the changes to the tables.
